docs/source/api/python/mantid/geometry/DetectorInfo.rst

```diff
@@ -0,0 +1,135 @@
+==============
+ DetectorInfo
+==============
+
+This is a python binding to the C++ class Mantid::Geometry::DetectorInfo.
+
+--------
+Purpose
+--------
+The purpose of the DetectorInfo object is to allow the user to access information about the detector(s) being used in an experiment. The DetectorInfo object can be used to access geometric information such as the number of detectors in the beamline, the absolute position of a detector as well as the absolute rotation of a detector.
+
+Many users may need this extra information so that they can have a better understanding of the beamline they are using. This extra information is also easy and fast to access meaning the users can make improvements to their experimental design with ease.
+
+DetectorInfo is one of three objects that the user can gain access to from a Workspace. 
+The other two are:
+  * SpectrumInfo
+  * ComponentInfo
+
+-------
+Usage
+-------
+
+**Example 1 - Creating a DetectorInfo Object:**
+This example shows how to obtain a DetectorInfo object from a workspace object.
+The return value is a ``DetectorInfo`` object.
+
+.. testcode:: CreateDetectorInfoObject
+	
+	# Create a workspace to use
+	ws = CreateSampleWorkspace()
+
+	# Get the DetectorInfo object
+	info = ws.detectorInfo()
+	print(type(info))
+
+**Example 2 - Calling the setMasked method on the DetectorInfo Object:**
+This example shows how to call the ``setMasked`` method.
+The method takes in an integer ``index`` parameter which corresponds to a component and a boolean ``masked`` parameter which allows the user to set the masking to True or False.
```

Comment:
We need to be **very precise** when we talk about what an `index` is. Not all `components` have a `index` that correspond to a detector. 

A **detector index** is a way of addressing and enumerating `Detectors` in the Beamline. Detector indexes are a continuous range and run from 0 to N-1 detectors. A Detector index can be found from a Detector ID using `indexOf`